src/spdx_tools/spdx3/model/security/cvss_v2_vuln_assessment_relationship.py

```diff
@@ -19,7 +19,7 @@
 
 @dataclass_with_properties
 class CvssV2VulnAssessmentRelationship(VulnAssessmentRelationship):
-    score: float = None
+    score: str = None
```

Comment:
Needs to be clarified as the markdown files in the spdx-3-model state `str` as well as `xsd:decimal`.
-> https://github.com/spdx/spdx-3-model/issues/332